### ISSUE TITLE
Add Helm Chart Documentation

### ISSUE THREAD

**Issue by defenestration** [kubereboot/website#46]
Would be nice to document, or at least link to, the helm chart in the https://github.com/kubereboot/charts repo.  I feel like it used to be in there but I did not see it currently.

**Comment by helloabhii** [comment]
Hey Alan B, Totally new here don't know how to work on this issue can you please guide me. I want to contribute and wanted to learn. I saw a lot of repositories issues but i didn't understand how to work on a issue.

**Comment by ckotzbauer** [comment]
Hi @helloabhii,
if you want to work on this issue, feel free to open a PR for it in this repository. In general you just have to look if someone is already working on an issue (it may be written in comments or the issue is assigned to somebody). If that's no the case, the issue is ready to be worked on, in general. If you have any questions you always can ping one of the maintainers in a comment: https://github.com/kubereboot/kured/blob/main/MAINTAINERS

**Comment by helloabhii** [comment]
Thanks for replying to me and also for the brief explanation @ckotzbauer,
I got what you said, But how do I work on this issue? What should I change in which part of the file? Please explain in detail. Again thanks !!!.

**Comment by ckotzbauer** [comment]
Hi @helloabhii,
for general guidance on contributing to open-source projects on GitHub, please follow these instructions: https://docs.github.com/en/get-started/quickstart/contributing-to-projects. In addition please read this section in out docs: https://kured.dev/docs/development/#certificate-of-origin

For the helm-chart documentation, please add a new section to this file: `content/en/docs/installation.md`

**Comment by helloabhii** [comment]
Thanks for replying @ckotzbauer,
I understand now how to contribute to projects,  But still, I didn't understand which section I have to add in the 'installation.md' file. Can you clarify more about what to add?

**Comment by ckotzbauer** [comment]
This issue is about the installation/usage of kured through a Helm-Chart. So add a section "Helm chart" and link to this document in a sentence below: https://github.com/kubereboot/charts/tree/main/charts/kured

**Comment by helloabhii** [comment]
Sorry for wasting your too much time, But  I don't know how to link these charts 
![image](https://github.com/kubereboot/website/assets/112479172/13fc2e56-3cf3-4a5a-a6d5-fa7d3dc55473)

What to do here.  Can you drop your DISCORD ID/link for more clarification? It will help me a lot, Thanks!

**Comment by ckotzbauer** [comment]
It is a [markdown format](https://www.markdownguide.org/basic-syntax/#links). You can also have a look at other files where links are already included.

**Comment by helloabhii** [comment]
@ckotzbauer, Sir do you use Discord or any other social handler where I can talk to you personally? Because am too dumb still didn't understand how to do all this.

**Comment by ckotzbauer** [comment]
No, I don't have a discord account. I adjusted the file in a way it could look like: [installation.md](https://github.com/kubereboot/website/files/13195866/installation.md)

Please compare the original file you checked out and my file to see which changes I made and decide if this is a version you want to submit as a PR or if you want to do further modifications.

**Comment by helloabhii** [comment]
![image](https://github.com/kubereboot/website/assets/112479172/5bebb832-1ab6-4d49-bf98-4800aeedb21f)
Oh, do I have to add only this line in the file called 'installation.md' ?

**Comment by ckotzbauer** [comment]
Yes, this is the new section I told you to add. I also added this additional headline at line 6, before the first paragraph starts:
## Default installation with manifests

**Comment by helloabhii** [comment]
![image](https://github.com/kubereboot/website/assets/112479172/edb90f52-1a6f-4a71-badc-05b468dbba77)
 
![image](https://github.com/kubereboot/website/assets/112479172/e9e19911-9324-426f-8f31-516f5ee51ac3)
 What is the issue now? Why merging is blocked?

**Comment by ckotzbauer** [comment]
@helloabhii I commented in the PR, the merge is blocked because no maintainer has approved and your commit is not signed-off (see my comment in the PR).

**Comment by helloabhii** [comment]
@ckotzbauer  Thanks for clarifying. I was busy with my exams. So I wasn't able to reply to you. I get it. Thanks!

**Comment by ckotzbauer** [comment]
Thanks for your work @helloabhii
